src/main/java/org/schemaspy/Revision.java

```diff
@@ -35,33 +37,21 @@
     }
 
     private static void initialize() {
-        InputStream in = null;
-        BufferedReader reader = null;
-
-        try {
-            in = Revision.class.getResourceAsStream(resourceName);
-
+        try (InputStream in = Revision.class.getResourceAsStream(resourceName)) {
             if (in != null) {
-                reader = new BufferedReader(new InputStreamReader(in));
+                try (BufferedReader reader = new BufferedReader(new InputStreamReader(in))) {
 
-                String line;
+                    String line;
 
-                while ((line = reader.readLine()) != null) {
-                    if (line.startsWith("Implementation-Build:")) {
-                        rev = line.split(" ")[1];
-                        break;
+                    while ((line = reader.readLine()) != null) {
+                        if (line.startsWith("Implementation-Build:")) {
+                            rev = line.split(" ")[1];
+                            break;
+                        }
                     }
                 }
             }
-        } catch (IOException exc) {
-        } finally {
-            try {
-                if (reader != null) {
-                    reader.close();
-                } else if (in != null) {
-                    in.close();
-                }
-            } catch (IOException ignore) {}
+        } catch (IOException ignored) {
```

Comment:
@danwatt please not leave this empty `catch ` section could you add something like:

```
logger.log(Level.SEVERE, ignored.getStackTrace().toString());
throw IOException;
```